daemon/src/command.rs

```diff
@@ -35,6 +35,20 @@ impl Executor {
         }
     }
 
+    /// A way of querying values of a CFD, without issuing any commands or
+    /// mutating its state.
+    ///
+    /// Can be useful when preparing a command.
+    pub async fn query<T>(&self, id: OrderId, query: impl Fn(&Cfd) -> Result<T>) -> Result<T> {
+        let cfd = self
+            .db
+            .load_open_cfd(id, ())
+            .await
+            .context("Failed to load CFD")?;
+
+        query(&cfd).context("Failed to execute command on CFD")
+    }
+
```

Comment:
I would like to bring this little thing to everyone's attention. I have added this method to be able to retrieve values more easily, when some additional data is needed. "query" or "peek" into the Cfd, without issuing a command.

Feedback welcome :-)